README.md

```diff
@@ -23,6 +24,229 @@ Terra Toolkit is a utility module used to facilitate independent development of
 
 - Install with [npm](https://www.npmjs.com): `npm install terra-toolkit --save-dev`
 
+## Webdriver.io Utility
+[Webdriver.io](http://webdriver.io/) is a framework for writing webdriver powered tests to validate functionality in browsers. Webdriver.io also provides easy services for setting up selenium, starting webpack and static servers, as well as accessibilty and visual regression testing.
+
+To make testing easier, Terar toolkit provides default configuration that enables the following:
+
+* Selenium docker instances for a consistent testing environment
+* Axe accessibility Utility
+* Visual regression testing
+* Viewport resizing helpers
```

Comment:
Is this what you meant by providing Viewport resizing helpers `https://github.com/cerner/terra-toolkit/pull/18/files#diff-04c6e90faac2675aa89e2176d2eec7d8R210`. I'd maybe reword this